tests/wea_test.py

```diff
@@ -33,6 +34,32 @@ def test_from_stat(self):
         assert wea_from_stat.location.city == 'Chicago Ohare Intl Ap'
         assert wea_from_stat.timestep == 1
 
+    def test_from_clear_sky(self):
+        """Test from original clear sky"""
+        stat_path = './tests/stat/chicago.stat'
+        wea_from_stat = Wea.from_stat_file(stat_path)
+        location = wea_from_stat.location
+        wea_from_clear_sky = Wea.from_ashrae_clear_sky(location)
+
+        assert wea_from_clear_sky.location.city == 'Chicago Ohare Intl Ap'
+        assert wea_from_clear_sky.timestep == 1
```

Comment:
We most likely want to check a number of values that are calculated as well as `city` and `timestep`.